include/bluetooth/mesh/main.h

```diff
@@ -157,6 +171,17 @@ struct bt_mesh_prov {
 	 */
 	void        (*complete)(u16_t net_idx, u16_t addr);
 
+	/** @brief Provisioning for a node is complete.
+	 *
+	 *  This callback notifies the application that provisioning has
+	 *  been successfully completed, and that a node has been assigned
+	 *  the specified NetKeyIndex and primary element address.
+	 *
+	 *  @param net_idx NetKeyIndex given during provisioning.
+	 *  @param addr Primary element address.
+	 */
+	void        (*complete_node)(u16_t net_idx, u16_t addr);
```

Comment:
Should there be an element count parameter here?

I'm also not completely happy with the name of the callback. Just brainstorming, but what would you think about `node_added`, `new_node` or `node_ready`?

Reply:
> Should there be an element count parameter here?

Yes I think it should be included.

> I'm also not completely happy with the name of the callback. Just brainstorming, but what would you think about `node_added`, `new_node` or `node_ready`?

`node_added` sounds good. Could even be `prov_db_node_added` to be clear but maybe it's a bit too long.

Reply:
Named it `node_added` and added the `num_elem` param,